service/controller/resource/clusterconfigmap/current.go

```diff
@@ -0,0 +1,43 @@
+package clusterconfigmap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/k8smetadata/pkg/label"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/cluster-apps-operator/pkg/project"
+	"github.com/giantswarm/cluster-apps-operator/service/controller/key"
+)
+
+func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) ([]*corev1.ConfigMap, error) {
+	cr, err := key.ToCluster(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	var configMaps []*corev1.ConfigMap
+	{
+		r.logger.Debugf(ctx, "finding cluster config maps in namespace %#q", key.ClusterID(&cr))
```

Comment:
I'm not sure how we logged this before, but better to concatenate two words.
```suggestion
		r.logger.Debugf(ctx, "finding cluster configmaps in namespace %#q", key.ClusterID(&cr))
```

Reply:
Yes, that's better. Thx!